test/unit/specs/components/common/SessionAccounts.spec.js

```diff
@@ -35,4 +35,10 @@ describe(`SessionAccounts`, () => {
   it(`renders the correct account`, () => {
     expect(wrapper.html()).toContain("Benjis account")
   })
+
+  it(`opens session modal and closes itself`, () => {
```

Comment:
The test description is off. This test doesn't test if anything closes.
```suggestion
  it(`links to the welcome screen`, () => {
```